packages/algob/src/lib/algo-operator.ts

```diff
@@ -208,7 +211,8 @@ export class AlgoOperatorImpl implements AlgoOperator {
       if (account.addr === creator.addr) {
         throw new BuilderError(ERRORS.SCRIPT.ASA_TRIED_TO_OPT_IN_CREATOR);
       }
-      const accountInfo = await this.algodClient.accountInformation(account.addr).do();
+      const accountInfo = await
+      this.algodClient.accountInformation(account.addr).do() as algosdk.modelsv2.Account;
```

Comment:
Could you explain why we're adding  `as algosdk.modelsv2.Account` here ? Previously it as not there.

Reply:
it's a compiler issue, when we load using do() function we get Record type which is used as Account type
Reference error:
`Argument of type 'Record<string, any>' is not assignable to parameter of type 'Account'.`